src/amo/api/collections.js

```diff
@@ -1,6 +1,10 @@
 /* @flow */
-import { callApi } from 'core/api';
+import { callApi, allPages } from 'core/api';
+import type {
+  ExternalCollectionAddon, ExternalCollectionDetail,
```

Comment:
This is just an aside but I think once we can't fit the entire import statement on one line we might as well do each one on a line; I think it's easier to scan the names if they're on newlines and it makes it easier to add/remove imports. 🤷🏻‍♂️